File: SeekMatch-Server/SeekMatch.Core/Entities/Education.cs
namespace SeekMatch.Core.Entities
{
    public class Education : BaseEntity
    {
        public required string Institution { get; set; }
        public string? Diploma { get; set; }
        public string? Domain { get; set; }
        public int StartYear { get; set; }
        public int StartMonth { get; set; }
        public int? FinishYear { get; set; }
        public int? FinishMonth { get; set; }
        public bool CurrentlyStudying { get; set; }
        public required Talent Talent { get; set; }
        public required string TalentId { get; set; }
    }

}

File: SeekMatch-Client/src/app/shared/models/education.ts
export class Education {

  public id: string | undefined = undefined;
  public institution: string = '';
  public diploma: string = '';
  public domain: string = '';
  public startYear : number = 0;
  public startMonth : number = 0;
  public endYear : number = 0;
  public endMonth : number = 0;
  public currentlyStudying : boolean = false;

  constructor(init?:Partial<Education>) {
    Object.assign(this, init);
  }
}
